1-js/02-first-steps/11-logical-operators/article.md

```diff
@@ -177,78 +177,78 @@ if (hour == 12 && minute == 30) {
 }
 ```
 
-Just as with OR, any value is allowed as an operand of AND:
+Assim como com OR, qualquer valor é permitido como um operando de AND:
 
 ```js run
-if (1 && 0) { // evaluated as true && false
+if (1 && 0) { // avaliado como verdadeiro && falso
   alert( "won't work, because the result is falsy" );
 }
 ```
 
 
-## AND finds the first falsy value
+## AND encontra o primeiro valor falso
 
-Given multiple AND'ed values:
+Dado vários valores AND'ed:
 
 ```js
 result = value1 && value2 && value3;
 ```
 
-The AND `&&` operator does the following:
+O operador AND `&&` faz o seguinte:
 
-- Evaluates operands from left to right.
-- For each operand, converts it to a boolean. If the result is `false`, stops and returns the original value of that operand.
-- If all operands have been evaluated (i.e. all were truthy), returns the last operand.
+- Avalia operandos da esquerda para a direita.
+- Para cada operando, converte para um booleano. Se o resultado for `false`, parará e retornará o valor original desse operando.
+- Se todos os operandos tiverem sido avaliados (isto é, todos foram verdadeiros), retorna o último operando.
 
-In other words, AND returns the first falsy value or the last value if none were found.
+Em outras palavras, AND retorna o primeiro valor falso ou o último valor, se nenhum foi encontrado.
 
-The rules above are similar to OR. The difference is that AND returns the first *falsy* value while OR returns the first *truthy* one.
+As regras acima são semelhantes a OR. A diferença é que AND retorna o primeiro valor *falso* enquanto OR retorna o primeiro valor *verdadeiro*.
 
-Examples:
+Exemplos:
 
 ```js run
-// if the first operand is truthy,
-// AND returns the second operand:
+// se o primeiro operando for verdadeiro,
+// AND retorna o segundo operando:
 alert( 1 && 0 ); // 0
 alert( 1 && 5 ); // 5
 
-// if the first operand is falsy,
-// AND returns it. The second operand is ignored
+// se o primeiro operando for falso,
+// AND o retorna. E o segundo operando é ignorado
 alert( null && 5 ); // null
 alert( 0 && "no matter what" ); // 0
 ```
 
-We can also pass several values in a row. See how the first falsy one is returned:
+Nós também podemos passar vários valores em uma linha. Veja como o primeiro falso é retornado:
 
 ```js run
 alert( 1 && 2 && null && 3 ); // null
 ```
 
-When all values are truthy, the last value is returned:
+Quando todos os valores são verdadeiros, o último valor é retornado:
 
 ```js run
-alert( 1 && 2 && 3 ); // 3, the last one
+alert( 1 && 2 && 3 ); // 3, o último
 ```
 
 ````smart header="Precedence of AND `&&` is higher than OR `||`"
-The precedence of AND `&&` operator is higher than OR `||`.
+A precedência do AND `&&` operator é maior que OR `||`.
```

Comment:
**A precedência do operador AND `&&` é maior que OR `||`.**
Forgot to translate the word **operator**